.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+package-lock.json
```

Comment:
This shouldn't be needed, as there's no `package-lock.json` in this repo.

Reply:
the problem is when I run `npm install` it gets added.

Reply:
I think your recommendation is always to use `yarn install` since you are providing the yarn.lock file. sorry I missed that.

Reply:
Yes, that would be the best given we have a `yarn.lock`.